index.html

```diff
@@ -0,0 +1,71 @@
+<html>
+
+<head>
+    <title>Nikos Village Websitelol</title>
+</head>
+
+<body>
+    
+    <div class="FadeIn"><h1>Welcome to Drosia!</h1></div>
+  
+    <div class="Fadein"><p>This is the test paragraph for the site. Maybe one day
+        it wont look like it was made in 1999.
+    </p></div>
```

Comment:
Formatting suggestion. VS code and many other tools usually offer a `code cleanup` option. I usually have mine set to run every time I save a file.

```suggestion
    <div class="Fadein">
      <p>
          This is the test paragraph for the site. Maybe one day
          it wont look like it was made in 1999.
      </p>
    </div>
```